Installing additional fluentd plugins in base image

dimitrovvlado opened the pull request:
Installing additional output plug-ins in base image:

- plugin for Apache Kafka > 0.8
- flowcounter plugin which counts messages/bytes that matches, per minutes/hours/days
- plugin which sends events to Amazon Kinesis
- plugin for mongoDB
- plugin to handle Facebook scribed thrift protocol
- plugin for mail
- plugin for influxdb
- plugin to count the number of matched messages, and emit if exceeds the threshold
- plugin for GrowthForecast

jvassev commented:
Thanks!
Let's limit the number of plugin you install to just:
* kafka
* thrift
* scribe
* email
* mongo

Also, kinesis is already installed. To make sure you don't accidentally install a plugin already there, sort the `gem install` commands in the Makefile.

Also mention the newly added plugins in the README.md file.

dimitrovvlado commented:
I've addressed the comments. According to the `README.md`, the `fluent-mixin-config-placeholders` plug-in should be installed, but I couldn't find that in the `Dockerfile`. Am I missing anything?
